file_transfer/receive_file.py

```diff
@@ -13,7 +13,7 @@
         print("Receiving data...")
         while True:
             data = sock.recv(1024)
-            print(f"data={data}")
+            print(f"data={str(data)}")
```

Comment:
Why this change?

Reply:
This and similar changes are redundant as string interpolation converts the object to string using the `__str__` method of the object.

Reply:
I'm not sure why but without it we get the following mypy-errors:
file_transfer/send_file.py:16: error: On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior
file_transfer/receive_file.py:16: error: On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior

Reply:
I guess it has something to do with the method from the socket-package that returns the data-object.

Reply:
`bytes` vs. `str` is the most time-consuming part of porting code from Python 2 to Python 3.
% __python3__
```
>>> type(b"abc")
<class 'bytes'>
>>> type("abc")
<class 'str'>
```